### ISSUE TITLE
data_loader DBLP

### ISSUE THREAD

**Issue by bobo18801737494** [safe-graph/DGFraud#29]
Can you explain the function load_data_dblp?

Why  do this data['net_APA'] - np.eye(N) ?

**Comment by YingtongDou** [comment]
The preprocessed DBLP data has already added self-loops for each subgraph.

In our code, we process the adjacency matrix using our function which can be found in [utils.py](https://github.com/safe-graph/DGFraud/blob/27865a576d76e4e5562a792dfc71b44d73e03c91/utils/utils.py#L8).

So we delete the existing self-loops to unify the data i/o format.

**Comment by bobo18801737494** [comment]
Thanks very much!
I have another question about train_test_split of  load_data_dblp,   split the adjacency matrix, may be lead to  nodes and realationship broken?

**Comment by YingtongDou** [comment]
The `train_test_split` only selects the nodes whose labels are used in the training phase or testing phase. It does not split the adj matrix.

During training, we leverage the relations and neighbors of the nodes in the training set to learn its embedding and predict its class.
